### ISSUE TITLE
Sagittarius changed its documentation layout in the sources

### ISSUE THREAD

**Issue by jpellegrini** [schemedoc/srfi-metadata#52]
The file that we were using, `sagittarius-scheme-master/doc/srfi.scrbl`, does not exist anymore:

```
$ ./generate.sh
...
head.sh
racket.sh
sagittarius-head.sh
tar: sagittarius-scheme-master/doc/srfi.scrbl: Not found in archive
tar: Exiting with failure status due to previous errors
```

**Comment by jpellegrini** [comment]
The commit didn't actually "fix" this issue; it only changed the autogenerated `.sh` file.

**Comment by jpellegrini** [comment]
This was fixed by PR #53
